kolibri/plugins/coach/assets/src/routes/index.js

```diff
@@ -47,6 +47,7 @@ export default [
     },
   },
   {
+    name: ClassesPageNames.HOME_PAGE,
```

Comment:
This creates an implicit coupling between the HomePage component name and this constant - it might be safer to use this constant to name the HomePage, to ensure that this is not accidentally broken by someone later deciding to rename the HomePage component to something else!

Reply:
I see what you're saying. And, is this different than some of the other ways that the page name constants are used throughout coach? i.e. [other class routes](https://github.com/marcellamaki/kolibri/blob/ebc5bea002ed58900d58f5e842f121394a10c7f9/kolibri/plugins/learn/assets/src/routes/classesRoutes.js#L29). Maybe it's not, and we are just trying to move away from that. I just used it for reference and am wondering if there is something specific here that is unique that we are trying to avoid.

Reply:
One thing we did differently in some coach routes was rely on the vue router implicit behaviour of using the component name as the route name. We are then making that implicit behaviour explicit, but wanted to make sure we don't make a footgun for ourselves.

Reply:
Ah, that makes sense. Thanks